azure/scripts/Set-ApimApiPolicy.ps1

```diff
@@ -37,7 +42,11 @@ try {
     Write-Host "Test that policy file exists"
     if (Test-Path -Path $ApimApiPolicyFilePath) {
         Write-Host "Set API policy"
-        Set-AzApiManagementPolicy -Context $ApimContext -Format application/vnd.ms-azure-apim.policy.raw+xml -ApiId $ApiId -PolicyFilePath $ApimApiPolicyFilePath -ErrorAction Stop -Verbose:$VerbosePreference
+        
+	$policy = Get-Content -Path $ApimApiPolicyFilePath -Raw
```

Comment:
If this create a variable with the ApplicationIdentifierUri replaced with the incoming variable - the is the line below correct using "-PolicyContent $content" would it not be $policy

Reply:
corrected